go_api/infrastructure/admin.go

```diff
@@ -0,0 +1,58 @@
+// DB操作
+package infrastructure
+
+import (
+	"github.com/NUTFes/lottery/go_api/domain"
+	"gorm.io/gorm"
+)
+
+type AdminInfrastructure struct {
+	db *gorm.DB
+}
+
+func NewAdminInfrastructure(db *gorm.DB) *AdminInfrastructure {
+	return &AdminInfrastructure{db: db}
+}
+
+// 全アドミンユーザーの取得
+func (u *AdminInfrastructure) FindAll() (*domain.Admins, error) {
+	admins := domain.Admins{}
+	if err := u.db.Find(&admins).Error; err != nil {
+		return nil, err
+	}
+	return &admins, nil
+}
```

Comment:
```suggestion
func (a *AdminInfrastructure) FindAll() (*domain.Admins, error) {
	admins := domain.Admins{}
	if err := a.db.Find(&admins).Error; err != nil {
		return nil, err
	}
	return &admins, nil
}
```
関数のレシーバーは構造体の頭文字の`a`や`admin`などの方が適切ですね。
例)`User`の時は`u`Adminの時は`e`

以下`Find`, `Create`, `Update`, `Delete`も修正お願いします